filebeat/_meta/fields.common.yml

```diff
@@ -150,3 +150,37 @@
       type: keyword
       description: >
         Name of organization associated with the autonomous system.
+
+    - name: kafka
+      type: group
+      fields:
+        - name: topic
+          type: keyword
+          description: >
+            Kafka topic
+
+        - name: partition
+          type: long
+          description: >
+            Kafka partition number
```

Comment:
@faec @urso I wonder if we should use the same fields metricbeat uses for this, in case we want to correlate and/or monitor the kafka queues used. Metricbeat uses `kafka.topic.name` for the topic and `kafka.partition.id` for the partition.

And I wonder if message specific fields (`offset`, `key`, `headers`) should be under an specific namespace as `kafka.message`, at least to differentiate the offset from other offsets.